.vscode/extensions.json

```diff
@@ -1,25 +1,17 @@
 {
     "recommendations": [
-        // "Required"
-        "ms-python.python",
         "davidanson.vscode-markdownlint",
         "stylelint.vscode-stylelint",
         "redhat.vscode-yaml",
-        "batisteo.vscode-django",
         "octref.vetur",
         "dbaeumer.vscode-eslint",
-        "EditorConfig.EditorConfig",
+        "editorconfig.editorconfig",
         "mikestead.dotenv",
-
-        // Recommended
         "esbenp.prettier-vscode",
         "lokalise.i18n-ally",
-        "eamodio.gitlens",
-        "Zignd.html-css-class-completion",
-        "reduckted.vscode-gitweblinks",
```

Comment:
Similar to gitlens, this feels more like a personal productivity tool, than something that's necessary for this particular project.